swagger.yml

```diff
@@ -151,6 +151,28 @@ paths:
           description: "Model not created"
           schema:
             type: "null"
+  /v1/model/read_latest_models:
```

Comment:
/v1/models/latest

Reply:
Not sure how connexion handles this, but will that collide with endpoint you previously propose as`/v1/models/{model_type}` ?

Reply:
It shouldn't, you would define /v1/models/latest as a constant and /v1/models/{model_type} has a url variable {model_type}, and I think it should recognize the difference, but I could definitely be wrong.   We could uses query parameters here.   Maybe uses /v1/models?latest=true?  And if latest=false we return all models (at least in theory, not sure we actually need to do that ever).

Reply:
Certainly worth a try! And if not I think your alternative should work